.circleci/config.yml

```diff
@@ -916,6 +916,45 @@ jobs:
       - run: *gitter_notify_failure
       - run: *gitter_notify_success
 
+  t_ems_compile_ext_ens:
+    docker:
+      - image: circleci/node:10
+    environment:
+      TERM: xterm
+    steps:
+      - checkout
+      - attach_workspace:
+          at: /tmp/workspace
+      - run:
+          name: Install test dependencies
+          command: |
+            sudo apt-get -qy install lsof
+      - run:
+          name: External Ens compilation
+          command: |
+            export COMPILE_ONLY=1
+            test/externalTests/ens.sh /tmp/workspace/soljson.js || test/externalTests/ens.sh /tmp/workspace/soljson.js
+
+  t_ems_test_ext_ens:
```

Comment:
Wondering why this job is not showing up in the CI. Also, the same for gnosis, colony and Zeppelin.

Reply:
Probably because of the missing workflow steps

Reply:
Looks line this one is still not running?

Reply:
Correct, running all the `test_ext_*` is part of #10397

Reply:
Why add the job here though?

Reply:
Good point. Removed